content/docs/start/experiments/index.md

```diff
@@ -14,6 +14,15 @@ ones that we need to Git.
 
 https://youtu.be/FHQq_zZz5ms
 
+<admon icon="fire">
+
+**New!** You can track and compare you ML experiments with DVC directly [from
+Visual Studio Code], a leading IDE in the industry.
+
+[from visual studio code]: /doc/vs-code-extension
```

Comment:
Unrelated but note: these relative links are redirected to https://marketplace.visualstudio.com/items?itemName=Iterative.dvc (actually the engine renders that external link directly so no redirect happens technically).

> This was in case we decided to write official docs for the extension in this site, which is still a possibility, I think.